### PR TITLE
Add dummy depthCompute lib generator

### PR DESCRIPTION
**Pull request by SeptimiuVana**
Signed-off-by: SeptimiuVana <Vana.Septimiu@analog.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -78,12 +72,17 @@ if (USE_ITOF)
         add_definitions(-DITOF)
 endif()
 
+if(WITH_DISABLE_DEPTH_LIBS)
```
Comment (dNechita) [inline CMakeLists.txt:75]: The variable name is kind of complicated. Maybe use: "USE_DEPTH_COMPUTE_STUBS".

File: apps/empty-library-generator/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 2.8)
+project(empty-library-generator)
```
Comment (dNechita) [inline apps/empty-library-generator/CMakeLists.txt:2]: Can we use a different name? The current one sounds like this project could generate lots of empty libraries. But, in fact, this is just a stub for depth compute library.
depth-compute-stub would be a much better name.

File: sdk/src/cameras/itof-camera/tofi/tofiComputeDefault.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 Analog Devices, Inc-> All Rights Reserved.
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/tofi/tofiComputeDefault.cpp:1]: why use "Default" suffix in the file name? Isn't tofiCompute.cpp good enough?

**Comment by dNechita** [comment]
I think that apps is not the right place for this. Try to put it here:
aditof-sdk-rework->common->adi
In the future, the common dir will look like this:
```
aditof-sdk-rework
|->common
      |->adi
            |->depth-compute-stub
      |->third-party
            |->json

```

File: CMakeLists.txt (hunk 1 above)
Comment (dNechita) [reply CMakeLists.txt:75]: This option should be added next to the rest of the rest of the options in the top level cmake. And documented.

File: apps/empty-library-generator/CMakeLists.txt (hunk 2 above)
Comment (SeptimiuVana) [reply apps/empty-library-generator/CMakeLists.txt]: Fixed.

File: CMakeLists.txt (hunk 1 above)
Comment (SeptimiuVana) [reply CMakeLists.txt]: Fixed.

File: sdk/src/cameras/itof-camera/tofi/tofiComputeDefault.cpp (hunk 3 above)
Comment (SeptimiuVana) [reply sdk/src/cameras/itof-camera/tofi/tofiComputeDefault.cpp]: Fixed.

File: common/adi/depth-compute-stub/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+add_library(tofi_compute SHARED ${LIB_SOURCES}/tofiCompute.cpp)
+set_target_properties(tofi_compute PROPERTIES
+    PUBLIC_HEADER ${LIB_SOURCES}/tofi_compute.h)
+
+add_library(tofi_config SHARED ${LIB_SOURCES}/tofiConfig.cpp)
+set_target_properties(tofi_config PROPERTIES
+    PUBLIC_HEADER ${LIB_SOURCES}/tofi_config.h)
+
+add_custom_command(TARGET ${PROJECT-NAME} PRE-BUILD
```
Comment (SeptimiuVana) [inline common/adi/depth-compute-stub/CMakeLists.txt:15]: correct form: add_custom_command(TARGET ${PROJECT_NAME} PRE_BUILD

**Review by dNechita — APPROVED** [review]

**Pull request merged**